.github/workflows/basic.yml

```diff
@@ -0,0 +1,96 @@
+name: Basic Tests and Documentation
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: "Default test set"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:    ["ubuntu-20.04"]
+        cxx:   ["g++"]
+        cc:    ["gcc"]
+        ver:   ["9"]
+        py:    ["3.6"]
```

Comment:
Should we bump this to 3.7?
It is time according to the numpy rules.

Reply:
Let's test anyway, it's nice to know when it starts failing.

Reply:
It should really say "minimum of whatever we support" in each category.
But that's hard to scrape form the website.